cli.js

```diff
@@ -46,7 +46,8 @@ const cli = meow(chalk`
   --no-hot            Instruct the client not to apply Hot Module Replacement patches
   --no-reload         Instruct middleware {italic not} to reload the page for build errors
   --open              Instruct the app to open in the default browser
-  --open-app          The name of the app to open the app within
+  --open-app          The name of the app to open the app within, or an array
+                      containing the app name and arguments for the app
```

Comment:
I think this argument requires more explanation. It needs to be clear this is a *JSON* array, not a JavaScript array (i.e. `['a', 'b', 'c',]` wouldn't work). Otherwise the console warning could be confusing to users.

Reply:
PR [here](https://github.com/webpack-contrib/webpack-serve/pull/73)